code/game/atoms_movable.dm

```diff
@@ -163,6 +163,7 @@
 /atom/movable/Move(atom/newloc, direct = 0, movetime)
 	if(!loc || !newloc) return 0
 	var/atom/oldloc = loc
+	setDir(direct)
```

Comment:
What is the purpose of this line? Breaks things like directional lock, and I am curious what other effects it can have, since this is *all* moveable objects, rather than just carbons

Reply:
This was in client/Move after Move proc called, because of that you were looking at the direction you're going
Firstly, i removed it because i thought inbuilt proc move that is called when we want to move something changes dir to the move direction and it worked fine until i started crawling. While crawling without this thing you were able to crawl with legs infront which is weird so i placed this thing here back. 
This should not break directional lock since directional lock is overrided in SetDir proc, but i will test it and find out the problem
I probably gonna place it in carbon/Move proc instead of this

Reply:
oh, i forgot to mention, i removed it from client/Move because it was breaking setting dir in Move proc, meaning when you hold neck grab and move, you will be in same direction as if you move in any other situations, that is why i removed it.

Reply:
fuck, i found why this is breaking forced look. Inbuild proc Move changes your direction as i said above and if you set dir before proc Move, inbuild proc overrides character dir
bit if you set dir after move, it means you cant change dir in Move proc like we change currently with neck grab

Reply:
the problem is fixed and the proc is moved to carbon/human in last update